docs/engine.md

```diff
@@ -102,6 +105,22 @@ engine.addRule(rule)
 
 //remove it
 engine.removeRule(rule)
+//or
+engine.removeRule(rule.id)
+```
+
+ ### engine.updateRule(Rule instance|Object options)
```

Comment:
don't forget to add this method to the table of contents, at the top of the file. that way it'll be discoverable

Reply:
@CacheControl I am not sure i understand what you mean, can you add commit suggestion?